Meta/Lagom/Tools/CodeGenerators/LibWeb/WrapperGenerator/IDLGenerators.cpp

```diff
@@ -560,7 +560,16 @@ static void generate_to_cpp(SourceGenerator& generator, ParameterType& parameter
     } else if (interface.enumerations.contains(parameter.type->name)) {
         auto enum_generator = scoped_generator.fork();
         auto& enumeration = interface.enumerations.find(parameter.type->name)->value;
-        enum_generator.set("enum.default.cpp_value", *enumeration.translated_cpp_names.get(optional_default_value.value_or(enumeration.first_member)));
+        StringView enum_member_name;
+        if (optional_default_value.has_value()) {
+            VERIFY(optional_default_value->length() >= 3 && (*optional_default_value)[0] == '"' && (*optional_default_value)[optional_default_value->length() - 1] == '"');
```

Comment:
Can't the default be `""`? XHR has a `""` enum, though it's not set as a default anywhere I know 🤔

Reply:
That's a good point, I'll make it `>= 2` instead